rebar.config

```diff
@@ -1,45 +1,39 @@
-%% -*- mode: erlang;erlang-indent-level: 2;indent-tabs-mode: nil -*-
-%% ex: ts=4 sw=4 ft=erlang et
+%% == Compiler and Profiles ==
+
 {erl_opts,
  [warn_unused_import, warn_export_vars, warnings_as_errors, verbose, report, debug_info]}.
 
 {minimum_otp_vsn, "23"}.
 
+{profiles,
+ [{test, [{cover_enabled, true}, {cover_opts, [verbose]}, {ct_opts, [{verbose, true}]}]}]}.
+
+{alias, [{test, [compile, format, hank, lint, xref, dialyzer, ct, cover]}]}.
+
+%% == Dependencies and plugins ==
+
+{project_plugins, [rebar3_hank, rebar3_hex, rebar3_format, rebar3_lint, rebar3_ex_doc]}.
+
+%% == Documentation ==
+
 {ex_doc,
  [{source_url, <<"https://github.com/inaka/katana-code">>},
   {extras, [<<"README.md">>, <<"LICENSE">>]},
   {main, <<"readme">>}]}.
 
 {hex, [{doc, #{provider => ex_doc}}]}.
 
-{project_plugins,
- [{rebar3_hex, "~> 7.0.1"},
-  {rebar3_format, "~> 1.1.0"},
-  {rebar3_lint, "~> 1.0.2"},
-  {rebar3_ex_doc, "~> 0.2.9"}]}.
```

Comment:
Not sure if it's worth keeping the version on these, since 1. they're working as is, 2. if something goes wrong we can either adapt or pin versions then...

Reply:
I very much prefer the pinned versions. I don't want PRs to _suddenly_ need lots of formatting/linting changes just because there is a new version of `rebar3_lint` around. I experienced that in NextRoll and that's why all NextRoll open-source tools, like the formatter and hank, have the versions pinned like this.
I believe it's a good practice, and to keep it up to date, we can periodically run `rebar3_depup` on the code, like NextRoll folks do with their repos and generate PRs _specifically_ to update the deps.

Reply:
I'll keep the versions: this pull request (part of a series that includes other lib.s with which you're involved) saw this `rebar.config` being the only one that had pinned versions, but this allows me to understand better what you prefer and I'll do the same for the other lib.s 😄

Reply:
This type of maintenance (which I seem to have done a lot in the past) kinda makes me think we'd benefit from a `rebar3` plugin that allowed to understand where your `rebar.config` don't respect certain rules (with some slack there's always a big part of `rebar.config` that we keep consistent in different projects), and also that there should probably be a centralised place (like a service) where we could apply depup to a bunch of lib.s (running e.g. once a day), instead of scattered in different repositories.

Reply:
a1550f9

Reply:
Rules for rebar.config files… that seems like an Elvis job to me. We already have _some_ rules there: https://github.com/inaka/elvis_core/blob/main/doc_rules/elvis_project/no_deps_master_rebar.md

Reply:
Maybe... I don't know if it fits stuff like "your `erl_opts` should have this and that, though", haven't really thought about the problem yet. In any case, if you start to get very different configurations from the default your config. itself would have to be shared. We'd have to find a small subset of "fixed" rules that we're most likely applying everywhere. (and we could also `elvis`-analyse the .gitignore, in that case, maybe (?)).